lib/Sema/TypeCheckStorage.cpp

```diff
@@ -1585,6 +1669,12 @@ synthesizeCoroutineAccessorBody(AccessorDecl *accessor, ASTContext &ctx) {
 
   bool isLValue = accessor->getAccessorKind() == AccessorKind::Modify;
 
+  if (isLValue &&
+      (storageReadWriteImpl == ReadWriteImplKind::StoredWithSimpleDidSet ||
+       storageReadWriteImpl == ReadWriteImplKind::InheritedWithSimpleDidSet)) {
+    return synthesizeModifyCoroutineBodyWithSimpleDidSet(accessor, ctx);
```

Comment:
@theblixguy Should we be checking for property wrappers / `@objc`, and avoid using `synthesizeModifyCoroutineBodyWithSimpleDidSet()` in those cases?

Reply:
Yeah, I think we'd need to suppress it for `@objc` properties. I am not sure about property wrappers though.